### DISCUSSION TITLE
Discuss how to use Discussions :-)

### DISCUSSION THREAD

**Discussion by vshcherb** [osmandapp/OsmAnd#10911] [General]
Hi all,

I think it's time to move on from  https://groups.google.com/g/osmand to Github Discussions - https://github.com/osmandapp/OsmAnd/discussions/. Actually we already did it for last 3 years, I think.

Most of our conversations is already happening in Github Issues. Sometimes we need to indicate that's not a bug and it's a rather discussion, exploring new ideas or something more like a question.

Discussions is not a documentation! For that purpose we've created https://docs.osmand.net/en and trying to contribute  daily improving our documentation. Though even initial documentation will take 3-6 months, it will not replace sharing  ideas and discussing internals process. In the end best from our discussions should come to Documentation eventually.

Welcome to Discussions,
Victor

**Comment by thompehw** [comment]
Hi! I've benefited from and contributed to the google group for a long time. I'm not opposed to the idea of changing but...

The google group / email list is clear that it's "Users helping users". And it has a nice ethos. No question is too stupid, and naive users are treated with courtesy. No one is flamed for not having searched the archives before asking their question.

I have noticed questions raised as github issues that should have been for "users helping users." But is that what github discussions is intended to be - users helping users, no question too naive or stupid? I suspect people may cling to the old user group while this is not clear.

Thanks!

**Reply by vshcherb** [reply]
well, google groups don't follow nice UI concept of Q&A, github issues are for actual work to be done and not Q&A. Groups could be used though anyway no restriction is imposed.

**Reply by barteis** [reply]
Is there a setting comparable to the Google Groups forum that highlights new  posts since the last visit?

**Comment by localjo** [comment]
As someone who's using OsmAnd mainly in an effort to get away from Google-everything, I appreciate moving to GitHub Discussions. I also prefer the GitHub UI, and think Discussions are intended to be for users helping users.

**Reply by thompehw** [reply]
Oh, yes, I can see the advantages and my remark was intended to be helpful. I was concerned that Github might be scary for people with naive questions especially if they aren't "computer people." But since the help links on osmand.net were updated to point here we are starting to see a full range of questions. As they accumulate, this will quickly look like a friendly place for people helping each other, and it'll be fine once the transition period is over.

**Reply by tommycrock** [reply]
Yeah, my main concern would be people having to sign up for yet another account being another barrier to participation

**Comment by tommycrock** [comment]
I have been receiving helpful comments via email from @thompehw to my Q&A (https://github.com/osmandapp/OsmAnd/discussions/11132) but don't know why they aren't showing up on the discussion board? It means my replies look like I'm replying to myself!

**Comment by thompehw** [comment]
I deleted two replies from the board! The first because I thought I may have made a mistake and I had no time left - I'm glad it helped to get to the bottom of this. The second because the XML I quoted didn't seem to appear when shown on the board, so the message made no sense!

**Reply by tommycrock** [reply]
Aah! mystery solved 😄
